mix.exs

```diff
@@ -42,7 +42,6 @@ defmodule Dawdle.MixProject do
       env: [
         backend: {:system, :module, "DAWDLE_BACKEND", Dawdle.Backend.Local},
         start_pollers: {:system, :boolean, "DAWDLE_START_POLLERS", false},
-        forward_raw_messages: {:system, :boolean, "DAWDLE_FORWARD_RAW_MESSAGES", false},
```

Comment:
why is this being removed?

Reply:
I migrated these changes from my own fork and accidentally included this.